source/request-as-event-emitter.js

```diff
@@ -1,4 +1,6 @@
 'use strict';
+/* istanbul ignore next: webpack only */
+const r = ({x: require})['yx'.slice(1)];
```

Comment:
This is only used in one place, so just put it inline with the function call at line 50.